lighthouse-core/test/audits/metrics-test.js

```diff
@@ -136,4 +136,25 @@ describe('Performance: metrics', () => {
     const result = await MetricsAudit.audit(artifacts, context);
     expect(result.details.items[0].interactive).toEqual(undefined);
   });
+
+  it('evaluates LayoutShift variants correctly', async () => {
+    const artifacts = {
+      traces: {
+        [MetricsAudit.DEFAULT_PASS]: clsAllFramesTrace,
```

Comment:
do we have a real-world test for these where they aren't all identical? if we don't know of any test sites, something that layoutshifts in a set pattern on glitch or something might be just as quick to spin up

Reply:
> do we have a real-world test for these where they aren't all identical? if we don't know of any test sites, something that layoutshifts in a set pattern on glitch or something might be just as quick to spin up

no, closest is `frame-metrics-m89`/`frame-metrics-m90`, but those both only have a single event in the main frame (the other 25 events in `frame-metrics-m89` are in other frames). I didn't really want to add another trace to our menagerie but I guess it's for the best :)

Reply:
added a trace